markdown/events/2020/hackathon-july-2020.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hackathon July 2020
+subtitle: our first fully remote hackathon
+type: hackathon
+start_date: "2020-07-13"
+start_time: "13:00"
+end_date: "2020-07-17"
+end_time: "17:00"
+---
+
+The nf-core hackathon in July this year will be fully virtual, so you
+can contribute to nf-core and get your pipeline started
+from the comfort of your homes.
+
+Join us to get help and guidance for writing your own Nextflow pipelines
+using the best practice guidelines implemented by the nf-core community
+or contribute to existing pipelines and advancing the current infrastructure.
+This will be a very informal event and it's great for learning,
+sharing ideas and solving issues intensively.
+
+## Schedule
```

Comment:
If the schedule is now fixed, we could add a Markdown table here with the full schedule.

https://www.tablesgenerator.com/markdown_tables